README.md

```diff
@@ -0,0 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> toDOList
+
+
+## Built With
+
+- HTML5. CSS3, Javascript, webpack, Linters
+
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+1. git clone git@github.com:oshanedesign/to-Do-List.git
+2. Cd into to-Do-List
```

Comment:
- [ ] Your live demo link is broken so it would be better to fix it or remove it completely since it serves no purpose.
- [ ] Also in the getting started section, kindly provide the necessary commands (`npm install`, `npm start`) that a user would need to set up and run your project locally 💯

Reply:
Thank you changes have been implemented.

live link : 
![live](https://user-images.githubusercontent.com/40554384/160158271-9e981db5-429f-4155-9f39-2957be95fea4.PNG)

Getting started: 
![start](https://user-images.githubusercontent.com/40554384/160158655-78ea34e9-f41d-4b02-812e-cb86d7825ed5.PNG)